cdap-ui/test/unit/filters/ellipsis-test.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+describe('myEllipsis', function() {
+
+  beforeEach(module('cdap-ui.filters'));
+
+  var ellipsis;
+  var $filter;
+
+  beforeEach(inject(function(_$filter_) {
+    $filter = _$filter_;
+  }));
+
+  beforeEach(function() {
+    ellipsis = $filter('myEllipsis');
+  });
+
+  it('should truncate string longer that limit to limit plus ellipsis', function() {
+
+    var input = 'string longer than ten characters';
+    var limit = 10;
+
+    expect(ellipsis(input, limit).length).toBe(limit + 1);
```

Comment:
May be if have ```ellipsis(input, limit)``` in a variable then again we can reduce the function to once instead of twice.